crates/forge/tests/integration/should_panic.rs

```diff
@@ -35,3 +38,43 @@ fn should_panic() {
 
     assert_passed!(result);
 }
+
+#[test]
+fn should_panic_unknown_entry_point() {
+    let test = test_case!(
+        indoc!(
+            r"
+            use array::ArrayTrait;
+            use starknet::{call_contract_syscall, ContractAddress, Felt252TryIntoContractAddress};
+            use result::ResultTrait;
+
+            use snforge_std::{declare, ContractClass, ContractClassTrait};
+
+            #[test]
+            #[should_panic]
```

Comment:
In rust, it's possible to have `should_panic` match expected error by a substring, meanwhile we [always assert the full message](https://github.com/foundry-rs/starknet-foundry/blob/07c3ef85a97bfb1ed8eacaa529f6e98569489f94/crates/forge-runner/src/test_case_summary.rs#L251). Maybe it's something we could add an option for?

Reply:
Here the string error cannot be asserted because it contains the address of a contract

Reply:
We can add an issue for that, good idea. I think substring should be the default behavior